duckduckgo/iOS
# Content blocking integration tests

Reviewer: Chris
Asana: https://app.asana.com/0/414709148257752/427082503331144

**Description**:
Add simple integration tests for content blocker counts. Note I have intentionally removed these from the main test suite so that they run manually. Once the content blocker work is complete and these tests pass we will add them again so that they run with CI.

**Steps to test his PR**:
1. Switch to the Integration test scheme
2. Run the Integration tests
3. Ensure that the python mock server is launched and the tests run. Only one test will pass - the others will fail until the content blocking work is complete. I have however validated that they pass on the spike branch.

###### Reviewer Checklist:


###### PR DRI Checklist:
